ConeSearch.tex

```diff
@@ -113,7 +113,7 @@ \subsection{Role within the VO Architecture}
 % Oh -- Notes don't need this; you'd have to remove archdiag.png
 % from FIGURES in the Makefile, too.
 
-\includegraphics[width=0.9\textwidth]{archdiag.png}
+\includegraphics[width=0.9\textwidth]{role_diagram.pdf}
 \caption{Architecture diagram for Simple Cone Search - TODO}
```

Comment:
Is the TODO still required after this change?

Reply:
Ops...you're right. Fixed!